src/ILCompiler.ReadyToRun/src/ObjectWriter/SectionBuilder.cs

```diff
@@ -53,27 +53,32 @@ public SymbolTarget(int sectionIndex, int offset)
     /// After placing an ObjectData within a section, we use this helper structure to record
     /// its relocation information for the final relocation pass.
     /// </summary>
-    public struct ObjectDataRelocations
+    public struct RelocatableBlock
```

Comment:
Bit of a nit but the name `RelocatableBlock` doesn't make it immediately obvious to me what this struct does (block is an overloaded term with basic block). Maybe something along the lines of `PlacedObjectData` to indicate it's `ObjectData` with some location information, in this case an offset?

Reply:
Good point, thanks; hopefully addressed in 2nd commit.

Reply:
Thanks - reads great 👍